docs/content/en/docs/architecture/components/scheduler/_index.md

```diff
@@ -1,13 +1,65 @@
 ---
-title: Keptn Lifecycle Scheduler
-linktitle: Scheduler
-description: Basic understanding of the Keptn Scheduler
+title: Keptn integration with Scheduling
+linktitle: Scheduler and Scheduling Gates
+description: Basic understanding of how Keptn integrates with Kubernetes Pod Scheduling
 weight: 80
 cascade:
 ---
 
+Keptn needs to integrate with Kubernetes scheduling to block
+the deployment of applications that do not satisfy Keptn defined pre-deployment checks.
+
+On Kubernetes versions older and including 1.26, Keptn uses a Scheduler to block applications deployment.
 The **Keptn Scheduler** is an integral component of Keptn that orchestrates
 the deployment process.
+
+From Keptn v0.8.4 onward, if you do have a K8s version greater than 1.26,
+Keptn
+uses
+the [Pod Scheduling Readiness K8s API](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-scheduling-readiness).
```

Comment:
please just talk about the current version and what it does/can do
we will have follow ups for the next version to talk about the default behaviour when it is fully enabled